src/preferenceCenter/preferenceCenter.blocks.js

```diff
@@ -0,0 +1,176 @@
+export default class PreferenceCenterBlocks {
+  blockManager;
+
+  constructor(editor) {
+    this.blockManager = editor.BlockManager;
+  }
+
+  addPreferenceCenterBlock() {
+    // add preference center category-list block
+    this.blockManager.add('MyCategories', {
+      tagName: 'div',
+      label: 'My Categories',
+      category: Mautic.translate('grapesjsbuilder.preferenceCenterLabel'),
+      draggable: false,
+      media:
+        '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M22 5.5c0-.3-.5-.5-1.3-.5H3.4c-.8 0-1.3.2-1.3.5v3c0 .3.5.5 1.3.5h17.4c.8 0 1.3-.2 1.3-.5v-3zM21 8H3V6h18v2zM22 10.5c0-.3-.5-.5-1.3-.5H3.4c-.8 0-1.3.2-1.3.5v3c0 .3.5.5 1.3.5h17.4c.8 0 1.3-.2 1.3-.5v-3zM21 13H3v-2h18v2z"/><rect width="10" height="3" x="2" y="15" rx=".5"/></svg>',
+      content: {
+        components: [
+          {
+            components: [{ type: 'label', components: 'My Categories' }],
```

Comment:
same here

Reply:
For My Category its the same as it was in legacy builder. I gave the same name for all labels from legacy builder.